examples/MyModule/CMakeLists.txt

```diff
@@ -132,22 +155,39 @@ execute_process(
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
+# Get the data install dir.
+execute_process(
+    COMMAND ${NEST_CONFIG} --datadir
+    RESULT_VARIABLE RES_VAR
+    OUTPUT_VARIABLE NEST_DATADIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Get the documentation install dir.
+execute_process(
+    COMMAND ${NEST_CONFIG} --docdir
+    RESULT_VARIABLE RES_VAR
+    OUTPUT_VARIABLE NEST_DOCDIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Get the library install dir.
+execute_process(
+    COMMAND ${NEST_CONFIG} --libdir
+    RESULT_VARIABLE RES_VAR
+    OUTPUT_VARIABLE NEST_LIBDIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
 # on OS X
 set( CMAKE_MACOSX_RPATH ON )
 
-# The module install directories in variables.
-set( INSTALL_BIN_DIR bin )
-set( INSTALL_INCLUDES_DIR include )
-set( INSTALL_LIB_DIR lib/nest )
-set( INSTALL_DATA_DIR share/nest )
-set( INSTALL_DOC_DIR share/doc/nest )
-
-# Complete paths of the install directories. Only used for the help installation.
-set( PKGSRCDIR ${PROJECT_SOURCE_DIR} )
-set( PKGBUILDDIR ${PROJECT_BINARY_DIR} )
-set( PKGDATADIR ${CMAKE_INSTALL_PREFIX}/${INSTALL_DATA_DIR} )
-set( PKGDOCDIR ${CMAKE_INSTALL_PREFIX}/${INSTALL_DOC_DIR} )
+# Install all stuff to NEST's install directories.
+set( CMAKE_INSTALL_LIBDIR ${NEST_LIBDIR}/nest CACHE STRING "object code libraries (lib/nest or lib64/nest or lib/<multiarch-tuple>/nest on Debian)" FORCE )
+set( CMAKE_INSTALL_DOCDIR ${NEST_DOCDIR} CACHE STRING "documentation root (DATAROOTDIR/doc/nest)" FORCE )
+set( CMAKE_INSTALL_DATADIR ${NEST_DATADIR} CACHE STRING "read-only architecture-independent data (DATAROOTDIR/nest)" FORCE )
```

Comment:
Yes. They should be relative paths. The full paths are created with `include( GNUInstallDirs )` and have the form `CMAKE_INSTALL_FULL_...`. The relative paths are used in the `install()` function as a destination within the `CMAKE_INSTALL_PREFIX`.